kornia/losses/dice.py

```diff
@@ -60,20 +67,32 @@ def dice_loss(input: torch.Tensor, target: torch.Tensor, eps: float = 1e-8) -> t
     if not input.device == target.device:
         raise ValueError(f"input and target must be in the same device. Got: {input.device} and {target.device}")
 
+    possible_average = ("micro", "macro")
+    if average not in possible_average:
+        raise ValueError(f"The `average` has to be one of {possible_average}. Got: {average}")
+
     # compute softmax over the classes axis
     input_soft: torch.Tensor = F.softmax(input, dim=1)
 
     # create the labels one hot tensor
     target_one_hot: torch.Tensor = one_hot(target, num_classes=input.shape[1], device=input.device, dtype=input.dtype)
 
+    # set dimensions for the appropriate averaging
+    dims = (2, 3)
```

Comment:
i think this will satisfy mypy (needs to check)
```suggestion
    dims: tuple[int, ...] = (2, 3)
```